tests/test_okf_frontmatter_and_footers.py

```
@@ -0,0 +1,102 @@
+class TestOKFFrontmatterAndFooters(unittest.TestCase):
+    """Validate OKF v0.1 frontmatter metadata and standard footers across docs."""
+
+    def test_docs_and_summary_okf_frontmatter(self) -> None:
+        """Verify that all markdown files in docs/ and root SUMMARY.md have valid OKF v0.1 frontmatter."""
+        target_files = [os.path.join(REPO_ROOT, "SUMMARY.md")]
+
+        docs_dir = os.path.join(REPO_ROOT, "docs")
+        for root, _, files in os.walk(docs_dir):
+            for file in files:
+                if file.endswith(".md"):
+                    target_files.append(os.path.join(root, file))
```

Ruff (0.16.1)

[warning] Use `list.extend` to create a transformed list

(PERF401)